index.bs

```diff
@@ -5651,9 +5660,34 @@ The <dfn export for=commands>emulation.setGeolocationOverride</dfn> command modi
 
 The [=remote end steps=] with |command parameters| are:
 
-1. If |command parameters| [=map/contains=] "<code>altitudeAccuracy</code>"
-   and |command parameters| doesn't [=map/contain=] "<code>altitude</code>",
-   return [=error=] with [=error code=] [=invalid argument=].
+1. If |command parameters| [=map/contains=] "<code>coordinates</code>"
+   and |command parameters| [=map/contains=] "<code>error</code>", return [=error=]
+   with [=error code=] [=invalid argument=].
+
+1. If |command parameters| doesn't [=map/contain=] "<code>coordinates</code>"
+   and |command parameters| doesn't [=map/contain=] "<code>error</code>", return
+   [=error=] with [=error code=] [=invalid argument=].
```

Comment:
This should be covered by the new group so it's not needed anymore.

Reply:
indeed, thanks